python/rpdk/java/templates/init/guided_aws/POJO.java

```diff
@@ -0,0 +1,24 @@
+// This is a generated file. Modifications will be overwritten.
+package {{ package_name }};
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+import java.util.List;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class {{ model_name|uppercase_first_letter }} {
```

Comment:
Please also include the recent fix for the POJO:  https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/commit/ddfbcbf34fc04b7b4ca63656be67e2514092598a